metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-corosync",
-  "version": "4.0.0",
+  "version": "4.0.1",
   "author": "voxpupuli",
```

Comment:
We decided a few weeks/days ago that we now stick with 'Vox Pupuli', can you update it please?